diff --git a/1.py b/1.py
new file mode 100644
index 0000000..3c7d064
--- /dev/null
+++ b/1.py
@@ -0,0 +1 @@
+312
\ No newline at end of file
diff --git a/index.html b/index.html
new file mode 100644
index 0000000..d01f779
--- /dev/null
+++ b/index.html
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    
+</body>
+</html>
\ No newline at end of file
diff --git a/new.txt b/new.txt
deleted file mode 100644
index e69de29..0000000
diff --git a/style.css b/style.css
new file mode 100644
index 0000000..ed3f455
--- /dev/null
+++ b/style.css
@@ -0,0 +1 @@
+f9c9e2c
\ No newline at end of file
